src/briefcase/integrations/subprocess.py

```diff
@@ -276,7 +276,7 @@ def _run_and_stream_output(self, args, check=False, **kwargs):
             self.stream_output(args[0], process)
             if process.stderr and kwargs["stderr"] != subprocess.STDOUT:
                 stderr = process.stderr.read()
-            return_code = process.poll()
+        return_code = process.poll()
```

Comment:
It is safer to call `poll()` after the exiting the `with` since `__exit__()` waits for the process to end.